src/fe/fe_boundary.C

```diff
@@ -38,23 +38,23 @@ namespace libMesh
 //-------------------------------------------------------
 // Full specializations for useless methods in 0D, 1D
 #define REINIT_ERROR(_dim, _type, _func)                        \
-  template <>                                                   \
+  template <>                                    \
   void FE<_dim,_type>::_func(const Elem *,                      \
                              const unsigned int,                \
                              const Real,                        \
                              const std::vector<Point> * const,  \
                              const std::vector<Real> * const)
 
 #define SIDEMAP_ERROR(_dim, _type, _func)                       \
-  template <>                                                   \
+  template <>                                    \
   void FE<_dim,_type>::_func(const Elem *,                      \
                              const Elem *,                      \
                              const unsigned int,                \
                              const std::vector<Point> &,        \
                              std::vector<Point> &)
 
 #define FACE_EDGE_SHAPE_ERROR(_dim, _func)                              \
-  template <>                                                           \
+  template <>                                            \
```

Comment:
No, thanks, the whitespace looks better aligned...

Reply:
That was a left-over from my previous attempt where I also modified template specializations (which turned out to be not required). Corrected in the updated PR